src/components/SignUp/ReviewCard.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import AwesomeSlider from 'react-awesome-slider';
+import 'react-awesome-slider/dist/styles.css';
+import './review.css';
+
+export const Slider = () => {
+  return (
+    <div className="review-container">
+      <AwesomeSlider organicArrows={false}>
+        <div className="review-card">
```

Comment:
Since this component is being re-used, maybe it could be turned into its own component and imported? If you want to make it more concise. But not needed, works fine as it is :)